Display/Documentation/HowToTurnOn.md

```diff
@@ -0,0 +1,28 @@
+How to turn on the Tetra Ventillator Splitter
+=============================================
+
+1. Ensure that the splitter is connected to a power supply
+
+2. Ensure that the airlines connecting the splitter and ventillator are attached (detailed in a serperate document)
```

Comment:
Are they called "airlines" in the specs/docs you've read?  Because I don't recall Daniel using that terminology.

Reply:
No I made that up.  If there's existing names I'll definitely change it.

Reply:
I've heard "tubing"

Reply:
Mark as resolved once there's an update in the code that's a fix :)

Reply:
I've now changed the language to tubing.